drakrun/requirements.txt

```diff
@@ -1,6 +1,4 @@
-karton-core==4.3.0
-minio==5.0.7
-redis==4.4.4
```

Comment:
is it that easy? were they just unused dependencies? Or just transitive dependencies from karton-core?